test/visibility/import/edgeCases/multipleDef.good

```diff
@@ -1,2 +1,2 @@
-multipleDef.chpl:9: error: symbol x is multiply defined
+multipleDef.chpl:2: error: symbol x is multiply defined
```

Comment:
The line number here (and below) should use that of the imports, since the imports are what is causing the conflict.  This would be solved by the map you avoided adding for the change.  I would be okay with tackling that in another commit, though